index.js

```diff
@@ -8,16 +8,54 @@ const data = require('./data');
 const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
 
 //Method 1 : Using Async Await
+// Drop the unique index on the title field
+
+
 
 const manageRecipes = async () => {
   try {
     // Connection to the database "recipe-app"
     const dbConnection = await mongoose.connect(MONGODB_URI);
     console.log(`Connected to the database: "${dbConnection.connection.name}"`);
 
+    
     // Before adding any recipes to the database, let's remove all existing ones
-    await Recipe.deleteMany();
+    // Drop the unique index on the title field
+    await Recipe.deleteMany({});
+    await Recipe.collection.dropIndexes();
+
+
+    const recipeDetail = {
+      title: 'Butter Muffin',
+      level: 'Easy Peasy',
+      ingredients: ['250g self-rasing flour', '1 tsp bicarbonate of soda', '150g chocolate chips', '100g golden caster sugar', '2 eggs', '150ml natural yogurt', '100g unsalted butter'],
+      cuisine: 'British',
+      image: 'https://images.immediate.co.uk/production/volatile/sites/30/2020/08/chocolate-chip-muffins-ccc8450.jpg?quality=90&webp=true&resize=440,400',
+      duration: 25,
+      creator: 'Lulu Grimes',
+    }
+    const recipeExample = await Recipe.create(recipeDetail);
+    console.log('Recipe is:', recipeExample.title)
+    // const dataRecipes = await Recipe.insertMany(data);
+    // dataRecipes.forEach((items) => console.log(items.title));
+    const uniqueData = Array.from(new Set(data.map((recipe) => recipe.title))).map((title) => data.find((recipe) => recipe.title === title));
```

Comment:
Gabriel, all this procedure just to `console.log` the titles of the recipe is unnecessary.
After you create the recipes using the `insertMany()` method and the `data` variable that you already have, you can just loop over the created recipes and `console.log` the title. You can use any kind of loop for this. I'll give you an example using a `for` loop.
```
 let dataRecipes = await Recipe.insertMany(data);
    // console log to see the titles of the inserted recipes
    for (let i = 0; i < dataRecipes.length; i++) {
      console.log(dataRecipes[i].title);
    }
```